app/src/main/java/app/grapheneos/camera/capturer/ImageCapturer.kt

```diff
@@ -36,25 +36,29 @@ class ImageCapturer(val mActivity: MainActivity) {
 
         cFadeAnim?.cancel()
 
-        cFadeAnim = AlphaAnimation(mActivity.captureButton.alpha, 0.6f)
-        cFadeAnim?.duration = 200
-        cFadeAnim?.interpolator = LinearInterpolator()
-        cFadeAnim?.fillAfter = true
+        AlphaAnimation(mActivity.captureButton.alpha, 0.6f).apply {
+            this.duration = 200
```

Comment:
`this` can be omitted on lines 40..42 and 55..57.
If you want to keep it for clarity (and for avoiding potential naming conflicts, which I support), replace `apply` with `let` and `this` with `it`.

Reply:
Yes using `let` should be a better option here since `this` could be confused with the instance of the main class (although had thought of skipping this then since `apply` is generally expected to be used in such cases)